themes/finna2/less/finna/public-favorites.less

```diff
@@ -18,10 +18,6 @@
         margin: 10px 0;
     }
 
-    .public-list-view {
-        margin-top: 20px;
-    }
-
```

Comment:
Tää vaikuttaa upotettuun listaan myös: jos on esim `<?=$this->userlistEmbed(['id' => 'x', 'allowCopy' => true])?>` niin nyt puuttuu margin listan ja kopiointinapin välistä